addon/README.md

```diff
@@ -1,5 +1,5 @@
 # Test Pilot
-The addon where ideas come to idea
+The add-on where ideas come to idea
```

Comment:
I left it for whimsy's sake.  Being in a readme is a lot different than someone seeing it in about:addons. :)